EnvChanger/Classes/EnvChangerController.swift

```diff
@@ -86,22 +144,25 @@ public class EnvChangerController<T>: UIViewController where T: EnvironmentRepre
         
         button.frame.size.width = newWidth
         button.frame.size.height = newHeight
-        button.imageEdgeInsets = UIEdgeInsets(top: imageEdgeInsets, left: imageEdgeInsets, bottom: imageEdgeInsets, right: imageEdgeInsets)
+        button.imageEdgeInsets = UIEdgeInsets(top: imageEdgeInsets,
+                                              left: imageEdgeInsets,
+                                              bottom: imageEdgeInsets,
+                                              right: imageEdgeInsets)
     }
     
     /// When initializing the controller with the given environments,
     /// saves the first found environment from the list to UserDefaults.
     private func saveFirstEnvironment() {
         guard let firstEnvironment = envs.allCases.first else { return }
         
-        if getSavedEnvironment() == "" {
-            UserDefaults.standard.set(firstEnvironment.environmentTitle, forKey: ACTIVE_ENV_KEY)
+        if savedEnvironment == "" {
```

Comment:
You can change it to `savedEnvironment.isEmpty`.

Reply:
The problem is that currently `savedEnvironment` is an optional. 

I could make it non-optional with a default value of `""` and then change the implementation here to use `.isEmpty` if that's cleaner?

Reply:
If it's optional, then it's okay to leave it like this.

Reply:
I actually think it's a bit cleaner as non-optional now :D. I've pushed it as non-optional, let me know what you think.

Reply:
Okay. Let it be non-optional :D